Core/OfficeDevPnP.Core/AppModelExtensions/FieldAndContentTypeExtensions.cs

```diff
@@ -739,6 +739,10 @@ public static void AddFieldToContentTypeByName(this Web web, string contentTypeN
         public static void AddFieldToContentType(this Web web, ContentType contentType, Field field, bool required = false, bool hidden = false)
         {
             contentType.EnsureProperties(c => c.Id, c => c.FieldLinks, c => c.SchemaXml);
```

Comment:
changing line 741 to be the following would be a cleaner fix:
contentType.EnsureProperties(c => c.Id, c => c.SchemaXml, c => c.FieldLinks, c => c.FieldLinks.Include(fl => fl.Id, fl => fl.Required, fl => fl.Hidden));